cognite/client/data_classes/contextualization.py

```diff
@@ -501,33 +511,25 @@ class VisionFeature(str, Enum):
 
     @classmethod
     def beta_features(cls) -> Set["VisionFeature"]:
-        return {VisionFeature.INDUSTRIAL_OBJECT_DETECTION, VisionFeature.PERSONAL_PROTECTIVE_EQUIPMENT_DETECTION}
+        """Returns a set of VisionFeature's that are currently in beta"""
+        return {cls.INDUSTRIAL_OBJECT_DETECTION, cls.PERSONAL_PROTECTIVE_EQUIPMENT_DETECTION}
 
 
+VISION_FEATURE_MAP: Dict[str, Any] = {
+    "text_predictions": TextRegion,
+    "asset_tag_predictions": AssetLink,
+    "industrial_object_predictions": ObjectDetection,
+    "people_predictions": ObjectDetection,
+    "personal_protective_equipment_predictions": ObjectDetection,
+}
+
+
+# Auto-generate the dataclass from the mapping VISION_FEATURE_MAP:
```

Comment:
I think this makes code completion engines useless for this class.

Reply:
Probably 🪄  Did you see the old auto-generate version? It parses type hints that has to be on a very specific format (and that will break once we move to "future" versions of annotations). Should I just write them out in full in both places?

Reply:
I'll write them both out in full, then make a test that ensures they have the same "members"